setup.py

```diff
@@ -5,9 +5,16 @@
     version='0.1.0',
     packages=['xbundle'],
     scripts=['bin/xbundle_convert'],
-    author='Isaac Chuang, Shawn Milochik',
-    author_email='ichuang@mit.edu, milochik@mit.edu',
+    author='MIT ODL Engineering',
+    author_email='odl-engineering@mit.edu',
     description='Converts edX courses between OLX and single-XML formats.',
     url='https://github.com/mitodl/xbundle',
     install_requires=['lxml', 'BeautifulSoup'],
+    license='BSD'
+    long_description=README,
```

Comment:
This is a python syntax error, you need to load the README into a variable ala: `README = open('README.rst').read()` or more properly:
```python
with open('README.rst') as readme_file:
    README = readme_file.read()
```